src/App.tsx

```diff
@@ -47,49 +62,110 @@ export function getReorderedGoods(
 //   sortType: SortType,
 // };
 
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+export class App extends Component<{}, ReorderOptions> {
+  state = {
+    sortType: SortType.NONE,
+    isReversed: false,
+  };
+
+  handleSortType = (order: SortType) => {
+    this.setState({ sortType: order });
+  };
+
+  handleOrderChange = () => {
+    this.setState((state) => ({
+      ...state,
+      isReversed: !state.isReversed,
```

Comment:
`State` destructuring is useless here

Reply:
fix @Anastasiia-Khudych comment
the process of copying all properties from state is processed inside `setState` function